src/components/digitalMedia/DigitalMedia.tsx

```diff
@@ -116,7 +120,7 @@ const DigitalMedia = () => {
     const DigitalMediaActions = (action: string) => {
         switch (action) {
             case 'json':
-                window.open(`https://sandbox.dissco.tech/api/v1/digitalmedia/${digitalMedia.id.replace('https://hdl.handle.net/', '')}`);
+                window.open(`https://dev.dissco.tech/api/v1/digitalmedia/${digitalMedia.id.replace('https://hdl.handle.net/', '')}`);
```

Comment:
This will keep being an issue, we need some way to inject ENV variables into the frontend, or at least set this dynamically. Could you make a task for this?

Reply:
Yes: https://naturalis.atlassian.net/browse/DD-698?atlOrigin=eyJpIjoiZDI0MGNjNjE2MzBiNDhiY2JkOTJjYzRlZTFhYzY2ZTUiLCJwIjoiaiJ9